docs/v3.x/installation/render.md

```diff
@@ -0,0 +1,35 @@
+# Render One-Click Deploy
+
+[Render](https://render.com) and Strapi make a great match. With persistent disks and managed PostgreSQL databases, Render gives you multiple different ways to store your content. Render services come with fully managed SSL, so it's no longer necessary to set up a proxy server to secure your Strapi app. Since Render services are automatically restarted if they become unresponsive, you don't need to use a process manager like `pm2` either.
+
+This guide explains how to deploy a new Strapi project on Render in one click. Visit [the Render deployment guide](../deployment/render.html) to learn how to update an existing Strapi project so it can be deployed on Render.
+
+::: tip
+For more information consult [Render's Deploy Strapi guide](https://render.com/docs/deploy-strapi)
+:::
+
+### Step 1: Create a Render Account
+
+Visit [the Render dashboard](https://dashboard.render.com) to create an account if you don't already have one.
+
+### Step 2: Fork an Example Repo
+
+Render maintains three Strapi on Render example repos, which differ based on which database is used and where uploaded media library files are stored:
+
+1. [Strapi with SQLite and uploads on disk](https://github.com/render-examples/strapi-sqlite)
+2. [Strapi with PostgreSQL and uploads on Cloudinary](https://github.com/render-examples/strapi-postgres-cloudinary)
+3. [Strapi with PostgreSQL and uploads on disk](https://github.com/render-examples/strapi-postgres)
+
+Read [Render's Deploy Strapi guide](https://render.com/docs/deploy-strapi) to get help choosing the best option for your use case. Once you've chosen, fork the repo on GitHub so you have the flexibility to make your own changes.
+
+### Step 3: Deploy
+
+Click the **Deploy to Render** button in the repo's `README` file. Give Render permission to access your repo if you haven't already. If you're using Cloudinary, you'll be prompted to enter your account credentials as environment variables.
```

Comment:
I can help with this, but should point out that a Deploy to Render button here would have to use one of our render-examples repos. If people fork the repo and then click the button in the README, Render will use their fork (it looks at the referrer) and they'll be able to deploy their own changes.

Maybe I should change it to:
```
Click the **Deploy to Render** button in your forked repo's `README` file.
```

Reply:
That might be a better option yeah. I'll check with our marketing team to see about getting a button but if you want to use a placeholder for now thats fine.

Reply:
Another option would be to have a **Deploy to Render** button that deploys Strapi in development mode. People would be able to experiment with creating their own content types, but would lose their changes when they re-deploy (because the new source files won't be committed). We'd emphasize in the doc that this is just for experimenting, not for production apps. What do you think?

Reply:
I'd suggest shying away from that generally, there is still a bit of confusion for some of our users about certain PaaS options (Heroku/GAE) about the dynamic nature of the deployment.

I think it's good to introduce these users into a proper source control development flow :)

Reply:
I updated Step 4 to focus more on introducing a source control development flow.